### PR TITLE
Improve performance by 66%

### PR DESCRIPTION
**Pull request by Hosch250**

### CODE REVIEW

File: RetailCoder.VBE/App.cs [hunk 1]
```diff
@@ -135,6 +135,11 @@ private void UpdateLoggingLevel()
 
         public void Startup()
         {
+            if (_vbe.VBProjects.Count != 0)
```
Comment (retailcoder) [inline RetailCoder.VBE/App.cs]: I'd rather start parsing after we're fully initialized. Here the menus aren't even created yet, and the configuration isn't loaded; I know it's not the case right now, but we might eventually introduce parsing options (e.g. "run inspections automatically" and whatnot), ...really doing this work before we're initialized doesn't seem like the best plan.

Comment (Hosch250) [inline RetailCoder.VBE/App.cs]: OK.

File: Rubberduck.VBEEditor/QualifiedModuleName.cs [hunk 2]
```diff
@@ -164,26 +85,6 @@ public QualifiedModuleName(VBComponent component)
         }
 
         /// <summary>
-        /// Creates a QualifiedModuleName for removing renamed declarations.
-        /// Do not use this overload.
-        /// </summary>
-        public QualifiedModuleName(VBComponent component, string oldComponentName)
```
Comment (retailcoder) [inline Rubberduck.VBEEditor/QualifiedModuleName.cs]: That overload smelled! Good riddance!

Comment (Hosch250) [inline Rubberduck.VBEEditor/QualifiedModuleName.cs]: It actually stopped being used when the sinks were refactored.

**Pull request merged**